src/app/client/src/app/modules/certificate/components/certificate-configuration/certificate-configuration.component.html

```diff
@@ -185,7 +185,7 @@ <h3>{{resourceService?.frmelmnts?.cert?.lbl?.noEditAction}}</h3>
               <div class="certificate-edit-sbcard">
                 <div class="d-flex flex-dc p-16 certificate-edit-sbcard-content">
                   <div class="certificate-frame">
-                    <div><img [src]="selectedTemplate?.artifactUrl"></div>
+                    <div><img src="{{selectedTemplate?.previewUrl || selectedTemplate?.artifactUrl}}"></div>
```

Comment:
Do reverse, if artifactUrl is not there then user previewUrl. 
Ideally we should not use previewUrl at all. This is only for release-3.3 we used.